python-avd/pyavd/_eos_designs/structured_config/underlay/dhcp_servers.py

```diff
@@ -60,13 +60,14 @@ def _update_subnets(self: AvdStructuredConfigUnderlayProtocol, dhcp_server: EosC
                     and "unnumbered" not in uplink.ip_address
                     and peer_facts.inband_ztp
                 ):
-                    subnet_item = EosCliConfigGen.DhcpServersItem.SubnetsItem(
+                    # ipv6 unumbered is not supported with inband_ztp hence right now only ipv4_subnet can be added
```

Comment:
the comment is correct but I am not sure this is a good explanation. Have you tried configuring IPv6 numbered subnets for inband ZTP (one uplink).

@ClausHolbechArista any insights?